tool/planet/etcd.go

```diff
@@ -196,9 +200,97 @@ func etcdEnable(upgradeService bool) error {
 		log.Info("etcd is in proxy mode, nothing to do")
 		return nil
 	}
+
 	return trace.Wrap(enableService(ctx, ETCDUpgradeServiceName))
 }
 
+// etcdInitJoin ensures this particular node is part of an etcd cluster.
+// Because the etcd cluster is re-created during an upgrade, this particular node may not be part of the new cluster.
```

Comment:
Could you elaborate on why this may be the case and why we didn't need to do this before?

Reply:
Before, it appears the nodes are able to rejoin, simply by being part of the initial cluster env variable, even though they're not members of the cluster, etcd would accept them. This is just supposition, as I didn't track down the exact behaviour change. 

After going to etcd 3.4, it seems to be more strict, and not allowing this style of join. Another possibility, is that by fixing the initial cluster status to existing, which we were incorrectly setting to a new cluster previously, was also allowing the nodes to join the cluster previously as an initial cluster config. Either way, the changes prevent a node from simply starting and joining the cluster, without a member object.